### DISCUSSION TITLE
Would it be possible to store a nested list of grouped tiddlers and flat list of tiddlers in the list field together?

### DISCUSSION THREAD

**Discussion by t5a** [TiddlyWiki/TiddlyWiki5#5890] [Q&A]
With a flat list, tiddler titles with no spaces can be listed with a space seperating them and multi-word titles get double square brackets. I want to store groups of tiddlers under category titles along with flat lists of tiddlers in the list field together. The nested lists could have a title followed by a colon and then followed by the inner list and maybe surrounded by braces {}.  Or is there an existing way of looking at this?

**Comment by AnthonyMuscio** [comment]
t5a

Yes there are other ways to handle the same requirement. However it has a lot to do with how you access the result, what you want to do with it. 

I think the easiest way is to store a list of filters, keeping in mind a tiddler title is the simplest form of a filter. So a list containing titles and filters can be iterated to a full list, ie the filters are the groups of tiddlers.  

tiddler [[tiddler 2]] [tag[tagname]] [[fieldname[fieldvalue]] 

However implied in what you are asking you may want to capture the list and groups at a moment in time eg "as at this point the following tiddlers and groups of tiddlers (ie the members of the groups) are as follows", if this is the case there is no need to retain the groups information, just store the whole list of tiddlers (with expanded groups).

To name a few other ways

- The list could be a list of tags with one or more other tiddlers they each tag.
- You are free to build the wiki-text and macros to do what you ask without it becoming built into tiddlywiki (you can control how the filed is handled - just avoid standard fields like tags/list/caption/description etc...
- you could have a list of macros or tiddlers to transclude where these macros or transclusions define a group of tiddlers
- you could have a list of tiddlers (groupnames) which have a tiddlers-list field containing a list of the members for each group

This request may be well served in the TiddlyWiki group forum, rather than here. But feel free to work through it in this discussion.  TRo me this is more an issue of the application of existing methods, ie "there is an [number of] existing way[s] to do this"

Regards
Tones/Tony
